platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -55,9 +57,10 @@ public static synchronized Mapbox getInstance(@NonNull Context context, @NonNull
     return INSTANCE;
   }
 
-  Mapbox(@NonNull Context context, @NonNull String accessToken) {
+  Mapbox(@NonNull Context context, @NonNull String accessToken, LocationSource locationSource) {
     this.context = context;
     this.accessToken = accessToken;
+    this.lostLocationSource = locationSource;
```

Comment:
Any reason why the local `locationSource` is called differently (`lostLocationSource `)?